python/pyspark/pandas/resample.py

```diff
@@ -481,20 +489,5 @@ def __getattr__(self, item: str) -> Any:
             else:
                 return partial(property_or_func, self)
 
-    def min(self) -> Series:
-        return first_series(self._downsample("min"))
-
-    def max(self) -> Series:
-        return first_series(self._downsample("max"))
-
-    def sum(self) -> Series:
-        return first_series(self._downsample("sum").fillna(0.0))
-
-    def mean(self) -> Series:
-        return first_series(self._downsample("mean"))
-
-    def std(self) -> Series:
-        return first_series(self._downsample("std"))
-
-    def var(self) -> Series:
-        return first_series(self._downsample("var"))
+    def _cleanup_and_return(self, psdf: DataFrame) -> Series:
+        return first_series(psdf).rename().rename(self._psser.name)
```

Comment:
qq: Why we do rename twice here?

Reply:
good question: I just copy it from `groupby`, let me check whether the first `rename` can be removed